javav2/example_code/sqs-jms/src/main/java/com/example/sqs/jms/spring/SpringExample.java

```diff
@@ -0,0 +1,117 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// snippet-start:[sqs-jms.java2.spring]
+package com.example.sqs.jms.spring;
+
+import com.amazon.sqs.javamessaging.SQSConnection;
+import com.example.sqs.jms.SqsJmsExampleUtils;
+import jakarta.jms.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.NoSuchBeanDefinitionException;
+import org.springframework.context.support.FileSystemXmlApplicationContext;
+
+import java.io.File;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+
+/**
```

Comment:
Not sure if this is a WIN issue. WHen i run this example, I get 

Exception in thread "main" org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from URL [file://C:/Users/scmacdon/BuildGit/aws-doc-sdk-examples-sqs_jms/javav2/example_code/sqs-jms/target/classes/SpringExampleConfiguration.xml.txt]
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:351)
	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:319)
	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:184)